client/src/Pages/NarratorWaitingScreen.jsx

```diff
@@ -1,17 +1,42 @@
 import React from 'react';
-import {Button} from 'antd';
+import {Button, Row, Col} from 'antd';
 
 export default class NarratorWaitingScreen extends React.Component{
 
     render(){
         return(
-            <div>
-                You are the Narrator...
-                <br/>
-                Press Start to begin the game
-                <br/> 
-                <Button>Start</Button>
-            </div>
+            <>
+            <Row style = {{paddingTop: "3vh", paddingBottom: "3vh"}}>
+                <Col xs = {0} md = {8}/>
+                <Col xs = {24} md = {8} style = {{textAlign: "center", fontFamily: "minecraft", fontSize: "200%"}}>
+                    You are the Narrator
+                </Col>
+                <Col xs = {0} md = {8}/>
+            </Row>
+            <Row style = {{paddingTop: "3vh", paddingBottom: "3vh"}}>
+                <Col xs = {0} md = {8}/>
+                <Col xs = {24} md = {8} style = {{textAlign: "center", fontFamily: "minecraft", fontSize: "200%"}}>
+                    Press Start to begin the game
+                </Col>
+                <Col xs = {0} md = {8}/>
+            </Row>
+            <Row style = {{paddingTop: "3vh", paddingBottom: "3vh"}}>
+                <Col xs = {0} md = {8}/>
+                <Col xs = {24} md = {8} style = {{textAlign: "center", fontFamily: "minecraft", fontSize: "200%"}}>
+                    (SELECT CARDS - TODO)
+                    {/* TODO - Add Model here with selection for each player card and such  */}
```

Comment:
I did start modifying my server code to support something like this. I'll document a WebSocket event to handle this in the future.